file-browser/src/components/SearchInput.js

```diff
@@ -5,6 +5,6 @@ import React from 'react';
 export default class SearchInput extends React.Component {
     render() {
         const props = this.props;
-        return (<div style={props.style}>SearchInput</div>)
+        return (<div style={props.style}>SearchInputs</div>)
```

Comment:
![image](https://user-images.githubusercontent.com/15992269/27258114-55dd0b7a-53a6-11e7-915f-80aa837a7fda.png)
Here is a screen shot. And the repo is https://github.com/jiamingguo/videosearch